requirements.txt

```diff
@@ -1,3 +1,18 @@
 wheel==0.41.2
 pip==23.3.1
 setuptools==68.2.2
+opentelemetry-instrumentation-aws-lambda==0.41b0
+opentelemetry-instrumentation-dbapi==0.41b0
+opentelemetry-instrumentation-logging==0.41b0
+opentelemetry-instrumentation-sqlite3==0.41b0
+opentelemetry-instrumentation-urllib==0.41b0
+opentelemetry-instrumentation-wsgi==0.41b0
+opentelemetry-instrumentation-flask==0.41b0
+opentelemetry-instrumentation-grpc==0.41b0
+opentelemetry-instrumentation-jinja2==0.41b0
+opentelemetry-instrumentation-pika==0.41b0
+opentelemetry-instrumentation-requests==0.41b0
+opentelemetry-instrumentation-sqlalchemy==0.41b0
+opentelemetry-instrumentation-tortoiseorm==0.41b0
+opentelemetry-instrumentation-urllib3==0.41b0
+
```

Comment:
The app dependencies should go only into `pyproject.toml`. We use `requirements.txt` only to kick start the builder image (see `Dockerfile`).

```suggestion
```